ui/src/utils/browser.ts

```diff
@@ -1,22 +1,11 @@
+import { AxiosError } from "axios";
 import { APIError, HTTPStatusError } from "src/adapters/api";
 
-export function downloadFile(blob: Blob, filename: string) {
```

Comment:
I removed it because I was doing house cleaning and this is not used anywhere.

The JSON is downloaded with `downloadJsonFromUrl` and didn't we agree to remove the download QR button?

Reply:
> The JSON is downloaded with downloadJsonFromUrl and didn't we agree to remove the download QR button?

Was this discussed and decided on at some point while I was away? I can't find mention of this apart from [this un-answered comment](https://www.figma.com/file/BE3tUbvtSB2EqmJsybpeFC?node-id=5792:165929#419746562), but all the designs remain unchanged.

Reply:
You will find more context [here](https://0xpolygon.slack.com/archives/C0409DDNV7V/p1681490246248899).

Reply:
That's why I am wondering:

> As proposed during the meeting we can include the download QR Code of a credential in the page credential details.

Reply:
Yes, if we finally decide to do that, we will have to add again this function.